libs/platform/i18n/src/lib/locale/default-locale.service.ts

```diff
@@ -28,18 +31,11 @@ export class DefaultLocaleService implements LocaleService {
     protected queryService = inject(QueryService)
   ) {
     this.active$ = concat(
-      this.adapter.getDefault().pipe(takeUntil(this.setActive$)),
+      this.adapter.getDefault().pipe(takeUntil(this.setActive$.pipe(skip(1)))),
```

Comment:
I think we could keep replay subject and then we should not need changes in this stream (skip(1) and filter)

Reply:
But in this case we will not be able to simply gat a value in `set` method. So it will a be a drade off in any case.